modules/claims_api/app/swagger/claims_api/v2/swagger.json

```diff
@@ -0,0 +1,599 @@
+  },
+  "tags": [
+    {
+      "name": "Veteran Identifier",
+      "description": "Allows authenticated veterans and veteran representatives to retrieve a veteran's id."
```

Comment:
nit: should Veteran be capitalized here? i feel like ive always seen it capitalized and i took that to be a respect type thing?

Reply:
Ah, I try to always capitalize those.  I must've missed it.
Will update.